examples/Dockerfile

```diff
@@ -14,7 +15,8 @@ ARG LOC=eidolon_examples
 FROM examples
 WORKDIR /app
 COPY $LOC .
-COPY demo_resources/ resources/
+COPY demo_resources/usage_resources.yaml resources/
```

Comment:
is this needed for the ui? We should make a todo to remove this requirement so the ui gracefully handles this being missing.

Reply:
Sort of, the problem was that the auth yaml was not working properly with the front end (that was the cause of the 403 forbidden errors I mentioned earlier). I made that change to only copy over usage_resources.yml so that the front end and back end could communicate properly